frontend/lib/norent/start-account-or-login/ask-phone-number.tsx

```diff
@@ -47,26 +51,34 @@ export const AskPhoneNumber: React.FC<StartAccountOrLoginProps> = (props) => {
           <>
             <PhoneNumberFormField
               {...ctx.fieldPropsFor("phoneNumber")}
-              label="Phone number"
+              label={li18n._(t`Phone number`)}
             />
             <div className="content">
-              <LetterBuilderAccordion question="Why do you need this information?">
-                We’ll use this information to either:
-                <ol className="is-marginless">
-                  <li>Log you into your existing account</li>
-                  <li>Match with a pre-existing account </li>
-                  <li>Sign you up for a new account.</li>
-                </ol>
+              <LetterBuilderAccordion
+                question={li18n._(t`Why do you need this information?`)}
+              >
+                <Trans id="norent.whyIsPhoneNumberNeeded">
+                  We’ll use this information to either:
+                  <ol className="is-marginless">
+                    <li>Log you into your existing account</li>
+                    <li>Match with a pre-existing account </li>
+                    <li>Sign you up for a new account.</li>
+                  </ol>
+                </Trans>
               </LetterBuilderAccordion>
               <p className="is-size-6">
-                Your privacy is very important to us! Everything on JustFix.nyc
-                is secure.{" "}
+                <Trans>
+                  Your privacy is very important to us! Everything on
+                  JustFix.nyc is secure.
+                </Trans>{" "}
                 <ModalLink
                   to={props.routes.phoneNumberTermsModal}
                   component={() => <PrivacyInfoModal />}
```

Comment:
Is internationalizing the PrivacyInfoModal out of scope here? I know this is a shared component between NoRent and the rest of the Tenant Platform so not sure how that would affect things exactly but just wanted to flag.